src/Access/IAccessStorage.cpp

```diff
@@ -747,14 +757,6 @@ void IAccessStorage::throwAddressNotAllowed(const Poco::Net::IPAddress & address
     throw Exception(ErrorCodes::IP_ADDRESS_NOT_ALLOWED, "Connections from {} are not allowed", address.toString());
 }
 
-void IAccessStorage::throwAuthenticationTypeNotAllowed(AuthenticationType auth_type)
-{
-    throw Exception(
-        ErrorCodes::AUTHENTICATION_FAILED,
-        "Authentication type {} is not allowed, check the setting allow_{} in the server configuration",
-        toString(auth_type), AuthenticationTypeInfo::get(auth_type).name);
-}
```

Comment:
May be that should still go the logs? 

BTW showing that exception to the user (i.e. how it is right now) disclosures some information about the user (i.e. the fact that it exists & have bad auth schema configured). 

But that information can be important for admins / DBAs.

Reply:
Added in the form of `LOG_INFO`